ee-feature-pack/galleon-common/src/main/resources/layers/standalone/iiop-openjdk/layer-spec.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="iiop-openjdk">
+    <dependencies>
+        <layer name="naming"/>
+    </dependencies>
+    <feature-group name="iiop-openjdk"/>
```

Comment:
This feature-group uses 'security="identity"' which AFAICT introduces a requirement for the legacy security subsytem. Which isn't ok; we don't want layers to require the legacy security subsystem

Reply:
If the layer instead specified 'security='elytron'" then a dependency on the 'elytron' layer would be required.

Such a dependency should be declared as optional, to allow the user to exclude the dependency if they intend to change the 'security' value.